yarn.lock

```diff
@@ -11582,7 +11582,7 @@ gzip-size@^6.0.0:
   dependencies:
     duplexer "^0.1.2"
 
-handlebars@^4.0.1, handlebars@^4.7.6:
```

Comment:
these seems suspicious, could use `npm ls handlebars` to checkout how these dependencies came about.

not a big deal, but worth cleanup if easy.